TauAnalysis/MCEmbeddingTools/plugins/EmbeddingVertexCorrector.cc

```diff
@@ -89,15 +87,6 @@ EmbeddingVertexCorrector::~EmbeddingVertexCorrector() {}
 // member functions
 //
 
-void EmbeddingVertexCorrector::beginRun(const edm::Run&, const edm::EventSetup& iEventSetup) {
-  // edm::ESHandle< SimBeamSpotObjects > beamhandle;
-  // iEventSetup.get<SimBeamSpotObjectsRcd>().get(beamhandle);
-
-  edm::ESHandle<BeamSpotObjects> beamhandle;
-  iEventSetup.get<BeamSpotObjectsRcd>().get(beamhandle);
-
-  edm::LogInfo("TauEmbedding") << "beam handle\n" << (*beamhandle);
```

Comment:
Consuming an ESProduct just for a INFO-level printout seemed potentially unnecessary to me, so I'm suggesting to remove it. If the printout is deemed necessary, I'll change the PR accordingly.